src/main/docker/gitlab/README.md

```diff
@@ -2,7 +2,7 @@
 
 First, configure the environment parameters:
 ```bash
-cp src/main/docker/env.example.gitlab-gitlabci.txt src/main/docker/.env
+cp src/main/docker/gitlab/gitlab-gitlabci.env.example src/main/docker/.env
```

Comment:
Should we just ignore it for now or cat it to the .env file to not override the new .env?